src/store/nativeServices/pushNotifications/pushNotifications.saga.ts

```diff
@@ -0,0 +1,19 @@
+import { AppRegistry, NativeModules, Platform } from 'react-native';
+import { call, take } from 'typed-redux-saga';
+import { pushNotifications } from '../../../services/pushNotifications/pushNotifications.service';
+
+export function* pushNotificationsSaga(): Generator {
+  const os = yield* take(Platform.OS);
```

Comment:
this works? I didn't expect take can be used like this

Reply:
No, it's not. But it's not working with 'call' either. I fixed it with next commit